modules/initiating-limited-live-migration.adoc

```diff
@@ -0,0 +1,88 @@
+
+.Procedure
+
+ifdef::openshift-rosa,openshift-dedicated[]
+. Enter the following command to add the `unsupported-red-hat-internal-testing` annotation to the cluster-level network configuration:
++
+[source,terminal]
+----
+$ oc patch Network.config.openshift.io cluster --type='merge' --patch '{"metadata":{"annotations":{"unsupported-red-hat-internal-testing": "true"}}}'
+----
+endif::[]
+
+. Enter the following command to patch the cluster-level networking configuration and initiate the migration from OpenShift SDN to OVN-Kubernetes:
++
+[source,terminal]
+----
+$ oc patch Network.config.openshift.io cluster --type='merge' --patch '{"metadata":{"annotations":{"network.openshift.io/network-type-migration":""}},"spec":{"networkType":"OVNKubernetes"}}'
+----
++
+After running this command, the migration process begins. During this process, the Machine Config Operator reboots the nodes in your cluster twice. It is expected that the migration takes approximately twice as long as a cluster upgrade.
++
+[IMPORTANT]
+====
+This `oc patch` command checks for overlapping CIDRs in use by OpenShift SDN. If overlapping CIDRs are detected, you must patch them before the limited live migration process can start. For more information, see "Patching OVN-Kubernetes address ranges".
+====
+
+. Optional: You can enter the following commands to ensure that the migration process has completed, and to check the status of the `network.config`:
++
+[source,terminal]
+----
+$ oc get network.config.openshift.io cluster -o jsonpath='{.status.networkType}'
+----
++
+[source,terminal]
+----
+$ oc get network.config cluster -o=jsonpath='{.status.conditions}' | jq .
+----
++
+You can check limited live migration metrics for troubleshooting issues. For more information, see "Checking limited live migration metrics".
+
+. Complete the following steps only if the migration succeeds and your cluster is in a good state:
+
+.. To remove the migration configuration from the `network.config` custom resource, enter the following command:
++
+[source,terminal]
+----
+$ oc patch Network.operator.openshift.io cluster --type='merge' \
+  --patch '{ "spec": { "migration": null } }'
+----
+
+.. To remove custom configuration for the OpenShift SDN network provider, enter the following command:
++
+[source,terminal]
+----
+$ oc patch Network.operator.openshift.io cluster --type='merge' \
+  --patch '{ "spec": { "defaultNetwork": { "openshiftSDNConfig": null } } }'
+----
+
+.. To remove the OpenShift SDN network provider namespace, enter the following command:
++
+[source,terminal]
+----
+$ oc delete namespace openshift-sdn
+----
+
+. After migration, you should check the metrics of EgressIP resources, EgressFirewall resources, and multicast enabled namespaces and ensure their intended behavior. See "Checking cluster resources after initiating the limited live migration" for more information.
```

Comment:
These steps are only needed by offline migration, not live migration.

Reply:
Will remove them, but we did include them in the docs that are live: https://docs.openshift.com/container-platform/4.16/networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.html#nw-ovn-kubernetes-live-migration_migrate-from-openshift-sdn